packages/core/src/js/feedback/FeedbackForm.types.ts

```diff
@@ -33,6 +33,12 @@ export interface FeedbackGeneralConfiguration {
    */
   showName?: boolean;
 
+  /**
+   * Should the screen shots field be included?
+   * Screen shots cannot be marked as required
+   */
```

Comment:
i: we should suggest the default set
```suggestion
   * @default false
   */
```